client/src/components/RatingsReviews/ReviewList.jsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable indent */
+import React, { useState } from 'react';
+import ReviewComponent from './ReviewComponent.jsx';
+import { useDispatch, useSelector } from 'react-redux';
+import actions from '../../state/actions/index.js';
+
+const ReviewList = (props) => {
+  const state = useSelector(state => state.reviews.metadataInfo);
```

Comment:
you should be pulling the productId out of state using the useSelector() hook instead of passing it down as a prop

```suggestion
const state = useSelector(state => {
  return {
    metadata: state.reviews.metadataInfo,
    productId: state.product.productId,
  }
}
```


And then you would need to update the places you called the variable 'state' to be state.metadataInfo.